integration_tests/test_server.py

```diff
@@ -1,7 +1,9 @@
+from __future__ import absolute_import
+from cloudify._compat import text_type
```

Comment:
This will only work on a `5.1` manager. best to create a local compat until 5.0.5 is no longer supported.